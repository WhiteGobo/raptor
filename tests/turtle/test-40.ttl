@prefix ns1: <http://example.com/> .
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .

[] ns1:prop _:Ne7d .

[] ns1:prop _:Ne7d .

_:N849 rdf:first ns1:b ;
    rdf:rest () .

_:Ne7d rdf:first ns1:a ;
    rdf:rest _:N849 .
