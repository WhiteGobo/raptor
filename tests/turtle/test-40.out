_:genid1 <http://example.com/prop> _:Ne7d .
_:genid2 <http://example.com/prop> _:Ne7d .
_:N849 <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> <http://example.com/b> .
_:N849 <http://www.w3.org/1999/02/22-rdf-syntax-ns#rest> <http://www.w3.org/1999/02/22-rdf-syntax-ns#nil> .
_:Ne7d <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> <http://example.com/a> .
_:Ne7d <http://www.w3.org/1999/02/22-rdf-syntax-ns#rest> _:N849 .
